### PR TITLE
SOCNotificationIntranetSpaceInvitationTestIt

### PR DESCRIPTION
**Pull request by balkissallem**

### CODE REVIEW

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/functional/SOCNotificationIntranetSpaceInvitationTestIt.java [hunk 1]
```diff
@@ -0,0 +1,494 @@
...
+        manageLogInOut.signIn(username1, password);
+        String status ="You're invited to join  " ;
+        navigationToolbar.goToIntranetNotification();
+        intranetNotification.goToAllNotification();
+        intranetNotification.checkStatusSpace(status, spaceName);
+        homePagePlatform.refreshUntil($(byText(spaceName)),Condition.visible,2000);
+
+        info("User A refuse invitation");
+        intranetNotification.refuseRqConnection(spaceName);
+        intranetNotification.checkNotStatusSpace(status, spaceName);
```
[the remark this answers is not in the document]
Comment (balkissallem) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/functional/SOCNotificationIntranetSpaceInvitationTestIt.java]: done

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/functional/SOCNotificationIntranetSpaceInvitationTestIt.java [hunk 2]
```diff
@@ -0,0 +1,494 @@
...
+    /**
+     * <li> Case ID:125144.</li>
+     * <li> Test Case Name: Accept a Space Invitation from notification.</li>
+     * <li> Pre-Condition: - User A is manager of the space 1
+     * - User A invite User B to join the space 1
+     * - Space Invitation notification is activated in the settings of User B</li>
+     * <li> Post-Condition: </li>
+     */
+    @Test
+    public void test02_AcceptASpaceInvitationFromNotification() {
```
[the remark this answers is not in the document]
Comment (balkissallem) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/functional/SOCNotificationIntranetSpaceInvitationTestIt.java]: Test case corrected

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/functional/SOCNotificationIntranetSpaceInvitationTestIt.java [hunk 3]
```diff
@@ -0,0 +1,494 @@
...
+		*Step Name: Step 2 : Check View all page
+		*Step Description:
+			- Click [Refuse]
+			- Go to View All
+		*Input Data:
+
+		*Expected Outcome:
+			- The notifications is not displayed / available in the View All page*/
+    @Test
+    public void test05_CheckViewAllAfterRefusingASpaceInvitation() throws Exception {
```
[the remark this answers is not in the document]
Comment (balkissallem) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/functional/SOCNotificationIntranetSpaceInvitationTestIt.java]: Test case corrected

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/functional/SOCNotificationIntranetSpaceInvitationTestIt.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (balkissallem) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/functional/SOCNotificationIntranetSpaceInvitationTestIt.java]: done

File: social/src/main/java/org/exoplatform/platform/qa/ui/social/pageobject/IntranetNotification.java [hunk 4]
```diff
@@ -454,33 +455,25 @@ public void acceptRqConnection(String fullName) {
...
       info("Click on Accept button");
       $(ELEMENT_NOTIFICATION_DROPDOWN).find(byText(fullName)).parent().parent().find(ELEMENT_BUTTON_ACCEPT_INVITATION).click();
-
     }
+
   }
 
+
+
+
```
[the remark this answers is not in the document]
Comment (balkissallem) [reply social/src/main/java/org/exoplatform/platform/qa/ui/social/pageobject/IntranetNotification.java]: corrected

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/functional/SOCNotificationIntranetSpaceInvitationTestIt.java [hunk 5]
```diff
@@ -0,0 +1,481 @@
...
+import static org.exoplatform.platform.qa.ui.core.PLFData.DATA_USER1;
+import static org.exoplatform.platform.qa.ui.selenium.Utils.getRandomNumber;
+import static org.exoplatform.platform.qa.ui.selenium.Utils.getRandomString;
+import static org.exoplatform.platform.qa.ui.selenium.locator.ActivityStreamLocator.ELEMENT_SPACE_MENU_ACTIVITY_PORTLET;
+import static org.exoplatform.platform.qa.ui.selenium.logger.Logger.info;
+
+
+@Tag("functional")
+@Tag("social")
+public class SOCNotificationIntranetSpaceInvitationTestIt extends Base {
```
Comment (ilyeskallala) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/functional/SOCNotificationIntranetSpaceInvitationTestIt.java]: class name must be "SOCNotificationIntranetSpaceInvitationTestIT" not "SOCNotificationIntranetSpaceInvitationTestIt"

Comment (balkissallem) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/functional/SOCNotificationIntranetSpaceInvitationTestIt.java]: Corrected

**Pull request merged**